reindexer/src/main/scala/uk/ac/wellcome/platform/reindexer/modules/ReindexModule.scala

```diff
@@ -1,29 +1,56 @@
 package uk.ac.wellcome.platform.reindexer.modules
 
+import javax.inject.Singleton
+
 import akka.actor.ActorSystem
 import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.google.inject.Provides
 import com.twitter.inject.{Injector, TwitterModule}
-import uk.ac.wellcome.utils.GlobalExecutionContext.context
+import uk.ac.wellcome.models.{CalmTransformable, MiroTransformable}
+import uk.ac.wellcome.platform.reindexer.ServerMain
+import uk.ac.wellcome.platform.reindexer.services._
 import uk.ac.wellcome.utils.TryBackoff
 
-object ReindexModule extends TwitterModule  with TryBackoff {
+object ReindexModule extends TwitterModule with TryBackoff {
```

Comment:
I _think_ TryBackoff might cause weird behaviour in this case, since it basically triggers a reindex every 100 milliseconds. We already have built in retry functionality. Having two types of retries I think it will only cause weird issues. What do people think?